kv.go

```diff
@@ -297,7 +297,14 @@ var (
 	ErrKeyDeleted             = errors.New("nats: key was deleted")
 	ErrHistoryToLarge         = errors.New("nats: history limited to a max of 64")
 	ErrNoKeysFound            = errors.New("nats: no keys found")
-	ErrKeyExists              = errors.New("nats: key exists")
+)
+
+// KeyValueError is an error result for KeyValue operations which internally
+// are JetStreamError as well so this is just an alias.
+type KeyValueError = JetStreamError
```

Comment:
Not sure we need this, we could just define `ErrKeyExists` as a `JetStreamError` probably

Reply:
I would opt for keeping it a `JetStreamError` as there may be many `JetStreamErrors` returned from KV operations anyway (e.g. `ErrStreamNotFound` etc.). Unless we want to only return `KeyValueErrors` from kv operations, I suppose we could keep it all as `JetStreamError`. Plus, there is nothing wrong with having the same error for object store, where `ErrKeyExists` would also make sense.

Reply:
Agree, I removed it for now.